index.html

```diff
@@ -1,192 +0,0 @@
-      <button><a href="./continue-reading.html">Continue Reading</a></button>
-    </section>
-
-    <hr>
-    <hr>
-    <!-- something of note with image and text float  -->
-    <section id="bottom">
-      <h3><a href="./continue-reading.html">Something Of Note</a></h3>
-      <img src="./images/fusion.jpg" alt=" " />
-      <h4>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Et iure qui tempora accusantium rerum voluptatem
-        molestias tenetur deleniti iusto provident, minima dicta, atque obcaecati laboriosam, est ratione adipisci
-        pariatur
-        quos nihil nostrum impedit suscipit nisi! Provident amet dolor eaque pariatur id! Repudiandae asperiores
-        cupiditate
-        pariatur.</h4>
-      <button><a href="./learn-more.html">Learn more</a></button>
-
-      <hr>
-
-
-      <h3><a href="./continue-reading.html">Something Of Less Note</a></h3>
-      <img src="./images/blackgoku.jpg" alt=" " />
-      <h4>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorem maxime dolor vero modi! Voluptatem dolores
-        tempore illum dolore dignissimos, eum, ipsam corporis blanditiis ex ab id! Porro, aliquid? Cum numquam sit
-        itaque,
-        nihil quas blanditiis. Placeat, facilis. Illum amet, ullam rerum quam et sed quas.</h4>
-      <button><a href="./learn-more.html">Learn more</a></button>
-    </section>
-  </section>
-
-  <!-- footer with contact info and questions or comments -->
-
-  <section id="footer">
-    <div class="container-footer">
-      <div class="contact row">
-        <h2>questions or comments? Get in touch</h2>
-
-
-        <div class="container-left">
-
-          <label for="name">Name</label>
-          <input type="text" id="name" name="name" required minlength="4" maxlength="8" size="20">
-
-          <label for="Email">Email</label>
-          <input type="text" id="email" name="name" required minlength="4" maxlength="50" size="20">
-
-          <label for="Message">Message</label>
-          <input type="text" id="message" name="name" required minlength="4" maxlength="100" size="50">
-          <button><a>Send Message</a></button>
-
-        </div>
-
-        <div class="container-right">
-
-          <h4>Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto doloremque, provident eaque, hic ea
-            voluptatum nobis facere rerum ex praesentium, cum magnam inventore! Optio recusandae magni ratione ea?</h4>
-          <h4>221B Baker Street London, England 00000 UK</h4>
-
-          <ul>
-            <li><a href="tel:">(000)123-4567</a></li>
-            <li><a href="mailto:">info@untitled.tld</a></li>
-            <li><a href="https://example.com">@untitled</a></li>
-            <li><a href="https://example.com">instagram.com</a></li>
-            <li><a href="https://example.com">dribbble.com</a></li>
-            <li><a href="https://example.com">facebook.com</a></li>
-          </ul>
-
-        </div>
-
-
-
-
-      </div>
-    </div>
-  </section>

```

Comment:
This section has everything it needs to be done it looks like. Getting the layout sorted a little bit more should wrap it up. You are very close as it sits right now!